### DISCUSSION TITLE
【绳网公告】如图

### DISCUSSION THREAD

**Discussion by AetherLude** [share121/inter-knot#2016] [其他]
![1](https://private-user-images.githubusercontent.com/76843407/357630860-78cdae03-724c-44da-bb5b-fbd23607941a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1OTY0NDQsIm5iZiI6MTcyMzU5NjE0NCwicGF0aCI6Ii83Njg0MzQwNy8zNTc2MzA4NjAtNzhjZGFlMDMtNzI0Yy00NGRhLWJiNWItZmJkMjM2MDc5NDFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDAwNDIyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFiYTg3ZGE3OWQ2OWQ3ZGZhMDA1Y2M2YWViMTg5MDFmNTVjNzg3MjdhMjI2M2JmYTlkZmNhNDJlNjE2NjcyYTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1aR2RkbYDL24fGRnjkZmpxyZVAmT9IF-UFhDo-3tyJU)

**Comment by TrafficLight6** [comment]
不是，哥们

**Comment by cpu644** [comment]
似了

**Comment by Myitian** [comment]
我贴被删了555

这图是随手拿画图P的
![图片](https://github.com/user-attachments/assets/f36b0b84-f0b3-42ee-a73a-e0e871b0ac9c)

**Reply by AetherLude** [reply]
> 我贴被删了555
> 
> 这图是随手拿画图P的 ![图片](https://private-user-images.githubusercontent.com/76843407/357645059-f36b0b84-f0b3-42ee-a73a-e0e871b0ac9c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1OTgxMTcsIm5iZiI6MTcyMzU5NzgxNywicGF0aCI6Ii83Njg0MzQwNy8zNTc2NDUwNTktZjM2YjBiODQtZjBiMy00MmVlLWE3M2EtZTBlODcxYjBhYzljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDAxMTAxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjNDQyYzM4OWIwYzBjYWMzOGY5ODYzYzk1ZGQzYjI0ZTQ3ZDUxY2IyNjQxNTczNWViZWQ3NWYzNzYyOGZhODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9oHhTu2huu3prjvAcjCEHeIG2r3bHtx-k7mIqJmUYKk)

图直接用的你上传到Github的图片（）

**Reply by Myitian** [reply]
> > 我贴被删了555
> > 这图是随手拿画图P的 ![图片](https://private-user-images.githubusercontent.com/76843407/357645059-f36b0b84-f0b3-42ee-a73a-e0e871b0ac9c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1OTgxMTcsIm5iZiI6MTcyMzU5NzgxNywicGF0aCI6Ii83Njg0MzQwNy8zNTc2NDUwNTktZjM2YjBiODQtZjBiMy00MmVlLWE3M2EtZTBlODcxYjBhYzljLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDAxMTAxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjNDQyYzM4OWIwYzBjYWMzOGY5ODYzYzk1ZGQzYjI0ZTQ3ZDUxY2IyNjQxNTczNWViZWQ3NWYzNzYyOGZhODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9oHhTu2huu3prjvAcjCEHeIG2r3bHtx-k7mIqJmUYKk)
> 
> 图直接用的你上传到Github的图片（）

我贴无了，封面也无了

**Reply by Myitian** [reply]
补档：
![](https://raw.githubusercontent.com/Myitian/-01/main/xibao.png)

**Comment by AetherLude** [comment]
# 玩原神去了

**Comment by AetherLude** [comment]
1

**Comment by match-fire** [comment]
# 我的VC1900！！

**Reply by inkluv** [reply]
我多少来着。

**Reply by match-fire** [reply]
> 我多少来着。

# 1750

**Reply by Buer-Nahida** [reply]
> 我多少来着。

我记得你有截图的吧（你@我 说你超过我的那次）

**Reply by match-fire** [reply]
1

**Reply by inkluv** [reply]
等轮换到我管理。给你补上。

**Comment by Buer-Nahida** [comment]
# 我的3290积分啊！！

**Comment by Buer-Nahida** [comment]
# 上一秒还听着音游曲呢

![image](https://github.com/user-attachments/assets/dd997f59-90d2-4101-ad62-85f0111e231e)

**Reply by inkluv** [reply]
这不是符玄大人吗。

**Reply by match-fire** [reply]
# 悲

**Reply by inkluv** [reply]
仙舟也有声云吗。

**Reply by Buer-Nahida** [reply]
> 仙舟也有声云吗。

我加进来的，频道主上线后给踢了

**Comment by capacitor1** [comment]
# 寄

**Comment by Buer-Nahida** [comment]
[开了一个Discord伺服器，暂时进来聊吧（在Va把绳网Discord重建之前）](https://discord.gg/tXKxUK4nSJ)
